crew

```diff
@@ -916,7 +930,7 @@ def archive_package (pwd)
   Dir.chdir CREW_DEST_DIR do
     system "tar c#{@verbose}Jf #{pwd}/#{pkg_name} *"
   end
-  system "sha256sum #{pwd}/#{pkg_name} > #{pwd}/#{pkg_name}.sha256"
+  File.write("#{pwd}/#{pkg_name}.sha512", Digest::SHA512.hexdigest( File.binread("#{pwd}/#{pkg_name}") ) )
```

Comment:
Don't we want csum and csum_type here also?

Reply:
Not needed, this is the part used to generate the checksum for packaged binary

Reply:
Not following.

Reply:
I mean this is a part only used in `crew build`, so it is fine to default `sha512sum`

Reply:
If you are going to do this then you might as well get rid of the other options for binary checksums and just go with sha512sum.

Reply:
Can we keep the compatibility for them? (since many source are still providing `sha256sum` in their checksum file)

Reply:
Sure but the way you have it set up currently, there is no way to implement md5 or blake2b checksums.

Reply:
Removed md5 support